ddl/ddl.go

```diff
@@ -194,6 +194,11 @@ type DDL interface {
 	SetHook(h Callback)
 	// GetHook gets the hook. It's exported for testing.
 	GetHook() Callback
+
+	// GetTableMaxRowID gets table max row ID. It's exported for testing.
+	GetTableMaxRowID(startTS uint64, tblInfo *model.TableInfo) (int64, bool, error)
+	// GetInformationSchema gets the infoschema binding to the ddl. It's expoted for testing.
+	GetInformationSchema() infoschema.InfoSchema
```

Comment:
Could we replace `GetInformationSchema()` with `domain.InfoSchema()` in test? Then we can remove this interface.